examples/C#/C#checker/C#checker.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Keep bounding with an older version if ok.

Reply:
only .Net 6.0 is installed on CI ENV.